### ISSUE TITLE
dependabot doesn't really work on this repo

### ISSUE THREAD

**Issue by johnabass** [xmidt-org/wrp-go#113]
We use ugorji's code generation on the various `wrp` message structs.  While this speeds up encoding and decoding, it does mean that we can't just update ugorji without rerunning the code generation.  See: https://ugorji.net/blog/go-codecgen, specifically "How do I know if I have to re-generate the code?".

Dependabot, being a general tool, doesn't know anything about this requirement.  That means that whenever dependabot thinks ugorji needs an update, all our unit tests for dependabot's PR fail.  Example: https://github.com/xmidt-org/wrp-go/pull/112

We should either (1) disable dependabot for this repo, or (2) figure out how to make dependabot smarter.

**Comment by schmidtw** [comment]
Good catch.  I think I'm missing a step during building where everything is regenerated.

**Comment by schmidtw** [comment]
We switched to msgp and this is not an issue any more.
